.github/workflows/iroha2-dev-pr.yml

```diff
@@ -137,3 +137,45 @@ jobs:
           file: Dockerfile
           # This context specification is required
           context: .
+
+  client-cli-test:
+    runs-on: ubuntu-latest
+    container:
+      image: alexstrokelive/iroha2-ci:pytest-nightly-2023-06-25
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build iroha_client_cli
+        run: |
+          cargo build --bin iroha_client_cli --release
+          cargo build --bin kagami --release
+          cargo build --bin iroha --release
```

Comment:
do you think we should run `release` or `debug` build?

Reply:
also note, that `debug` is faster to build

Reply:
Ok, debug is really faster. I leave debug option for it.